### DISCUSSION TITLE
Refactor OrchardCore.Twitter to support API v2

### DISCUSSION THREAD

**Discussion by davidpuplava** [OrchardCMS/OrchardCore#15616] [Ideas]
Is anyone working on updating OrchardCore.Twitter to support v2 of X's API?

I see @hishamco has an open PR to rebrand Twitter to X, and @MichaelPetrinolis looks to be the original developer on this module. If nobody is working on this, I can try and submit a PR for it. I'd also add workflow activities to support the Manage Tweets and Users lookup endpoints in the "Free" tier.

**Comment by hishamco** [comment]
I already did PR if I'm not wrong a long time ago but seems I closed. @davidpuplava feel free to open your PR, I will finalize the branding PR then you can rebase

**Reply by davidpuplava** [reply]
Thanks @hishamco - I see the branding PR is merged in. I'll pull/rebase and submit a PR for my API v2 change.

**Reply by hishamco** [reply]
Waiting for your PR ..
